java/README.md

```diff
@@ -60,7 +60,7 @@ Lite runtime](lite.md).
 
 Bazel has native build rules to work with protobuf. For Java, you can use the
 `java_proto_library` rule for server and the `java_lite_proto_library` rule
-for Android. Check out [our build files examples](../examples/BUILD) to learn
+for Android. Check out [our build files examples](../examples/BUILD.baze) to learn
```

Comment:
Looks like you missed the final `l` in .bazel

Reply:
My bad. Thank you! Fixed!